telegram/ext/updater.py

```diff
@@ -370,34 +396,44 @@ def _check_ssl_cert(self, cert, key):
     def _gen_webhook_url(listen, port, url_path):
         return 'https://{listen}:{port}{path}'.format(listen=listen, port=port, path=url_path)
 
-    def _bootstrap(self, max_retries, clean, webhook_url, allowed_updates, cert=None):
-        retries = 0
-        while 1:
+    def _bootstrap(self, max_retries, clean, webhook_url, allowed_updates, cert=None,
+                   bootstrap_interval=5):
+        retries = [0]
```

Comment:
Why did you make this a list? It's still only used for one int value

Reply:
Because otherwise it will overridden as a local variable of the inner function.
A different workaround would be to do:
```python
global retries
retries = 0
```
But I don't like `global` inside such an inner code.
Of course we can do it a class method or something more "generic" but then we'll lose code locality. 
If you can think of something else, I'll be happy for a more elegant solution.